pytorch_blade/src/torch-mlir/include/torch-mlir/Conversion/TorchToMhlo/TorchToMhlo.h

```diff
@@ -27,16 +27,14 @@
 
 namespace mlir {
 class ModuleOp;
-
 namespace torch {
-
-std::unique_ptr<OperationPass<func::FuncOp>> createConvertTorchToMhloPass();
-
 namespace Torch {
```

Comment:
Any rules on the namespaces? `mlir::torch::Torch` confused me ...

Reply:
Following the TorchMLIR naming convention. Torch means Torch Dialect, TorchConversion means TorchConversion Dialect.